### PR TITLE
Add visualization docs

### PR DESCRIPTION
**Pull request by hhaAndroid**
## Motivation

Reference from https://github.com/open-mmlab/mmocr/blob/1.x/docs/en/user_guides/visualization.md

### CODE REVIEW

File: docs/en/user_guides/visualization.md [hunk 1]
```diff
@@ -1 +1,90 @@
 # Visualization
```
Comment (RangiLyu) [inline docs/en/user_guides/visualization.md:1]: Check for duplicates with user_guides/useful_tools.html#visualization

File: docs/en/user_guides/visualization.md [hunk 2]
```diff
@@ -1 +1,90 @@
...
+
+## Storage
+
+MMDet uses the local visualization backend [`LocalVisBackend`](mmengine.visualization.LocalVisBackend) by default, and the model loss, learning rate, model evaluation accuracy and visualization The information stored in `VisualizerHook` and `LoggerHook`, including loss, learning rate, evaluation accuracy will be saved to the `{work_dir}/{config_name}/{time}/{vis_data}` folder by default. In addition, MMDet also supports other common visualization backends, such as `TensorboardVisBackend` and `WandbVisBackend`, and you only need to change the `vis_backends` type in the configuration file to the corresponding visualization backend. For example, you can store data to `TensorBoard` and `Wandb` by simply inserting the following code block into the configuration file.
+
+```Python
+_base_.visualizer.vis_backends = [
+    dict(type='LocalVisBackend'),
+    dict(type='TensorboardVisBackend'),
+    dict(type='WandbVisBackend'),]
```
Comment (RangiLyu) [inline docs/en/user_guides/visualization.md:42]: Add an introduction to these backends' settings, or add links to their api documentation

File: docs/en/user_guides/visualization.md [hunk 3]
```diff
@@ -1 +1,90 @@
...
+
+- Implement common drawing APIs, such as [`draw_bboxes`](mmengine.visualization.Visualizer.draw_bboxes) which implements bounding box drawing functions, [`draw_lines`](mmengine.visualization.Visualizer.draw_lines) implements the line drawing function.
+- Support writing visualization results, learning rate curves, loss function curves, and verification accuracy curves to various backends, including local disks and common deep learning training logging tools such as [TensorBoard](https://www.tensorflow.org/tensorboard) and [Wandb](https://wandb.ai/site).
+- Support calling anywhere in the code to visualize or record intermediate states of the model during training or testing, such as feature maps and validation results.
+
+Based on MMEngine's Visualizer, MMDet comes with a variety of pre-built visualization tools that can be used by the user by simply modifying the following configuration files.
+
+- The `tools/analysis_tools/browse_dataset.py` script provides a dataset visualization function that draws images and corresponding annotations after Data Transforms, as described in [`browse_dataset.py`](useful_tools.md).
```
Comment (RangiLyu) [inline docs/en/user_guides/visualization.md:13]: Add link to the doc of browse_dataset in 'useful_tools'
